cmd/azure.go

```diff
@@ -151,7 +151,7 @@ func (a AzureObjects) StorageInfo() StorageInfo {
 }
 
 // MakeBucket - Create a new container on azure backend.
-func (a AzureObjects) MakeBucket(bucket string) error {
+func (a AzureObjects) MakeBucket(location, bucket string) error {
```

Comment:
you can override MakeBucket in gateway-interface as location does not make sense for fs/xl